src/mame/drivers/segaxbd.cpp

```diff
@@ -316,9 +316,25 @@ segaxbd_state::segaxbd_state(const machine_config &mconfig, device_type type, co
 	palette_init();
 }
 
+void segaxbd_state::install_segapcm(const char *region)
+{
+	// lowest segapcm bank bit is unused
+	uint32_t len    =   memregion(region)->bytes();
+	uint8_t *src    =   memregion(region)->base();
+	std::vector<uint8_t> buf(len);
+
+	for (int i = 0; i < len / 2; i += 0x10000)
+	{
+		memcpy(&buf[(i * 0x20000)],           src[i * 0x10000], 0x10000);
+		memcpy(&buf[(i * 0x20000) + 0x10000], src[i * 0x10000], 0x10000);
+	}
+	memcpy(&src[0], &buf[0], len);
+}
```

Comment:
This screams that it's a hack.  Are you sure you can't do this properly with address maps?